fastlane_bot/helpers/txhelpers.py

```diff
@@ -7,23 +7,20 @@
 __VERSION__ = "1.0"
 __DATE__ = "01/May/2023"
 
-import asyncio
-import nest_asyncio
 from _decimal import Decimal
 
-from json import loads
 from dataclasses import dataclass
-from typing import List, Any, Dict, Optional
+from typing import List, Any, Dict
 
-import requests
 from alchemy import Network, Alchemy
 from web3.exceptions import TimeExhausted
 
 from fastlane_bot.config import Config
 from fastlane_bot.data.abi import ERC20_ABI
-from fastlane_bot.utils import num_format, log_format, num_format_float, int_prefix
+from fastlane_bot.utils import num_format, log_format
 
-nest_asyncio.apply()
+MAX_UINT256 = 2 ** 256 - 1
+ETH_DECIMALS = 10 ** 18
```

Comment:
I wanted to avoid the long name `ETH_RESOLUTION`, and I was hoping that no one would notice :)

Please suggest a name if you can think of something better than `ETH_RESOLUTION`.

Reply:
ETHDECIMALSMULTIPLIER